src/main/java/org/cactoos/text/TextOf.java

```diff
@@ -341,6 +348,50 @@ public TextOf(final InputStream input) {
         this(new InputOf(new InputStreamReader(input, StandardCharsets.UTF_8)));
     }
 
+    /**
+     * Formats date using ISO date time format.
+     * @param date The date to format.
+     * @since 1.0
+     */
+    public TextOf(final LocalDate date) {
+        this(date, new Iso().value());
+    }
+
+    /**
+     * Formats date using provided date time format string using default locale.
+     * @param date The date to format.
+     * @param format The format to use.
+     * @since 1.0
+     */
+    public TextOf(final LocalDate date, final String format) {
+        this(date, format, Locale.getDefault(Locale.Category.FORMAT));
+    }
+
+    /**
+     * Formats the date using the provided format string using the provided
+     * locale.
+     * @param date The date to format.
+     * @param format The format string to use.
+     * @param locale The locale to use.
+     * @since 1.0
+     */
+    public TextOf(final LocalDate date, final String format,
+        final Locale locale) {
+        this(date, DateTimeFormatter.ofPattern(format, locale));
+    }
+
+    /**
+     * Formats the date using the provided formatter.
+     * @param date The date to format.
+     * @param formatter The formatter to use.
+     * @since 1.0
+     */
+    public TextOf(final LocalDate date, final DateTimeFormatter formatter) {
+        this(() -> formatter.format(
+            ZonedDateTime.of(date, LocalTime.MIN, ZoneId.systemDefault())
```

Comment:
@Iprogrammerr since `LocalDate` and `DateTimeFormatter` are both immutable, I believe it would make sense to envelope this into a `StickyScalar`

Reply:
@victornoel @Iprogrammerr not really required in this case, but I'll allow it